### PR TITLE
Updown checks 2

### PR DESCRIPTION
**Pull request by gestchild**
Script to add/delete checks

### CODE REVIEW

File: updown/webapp/utils.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+function checksAreEqual(check1, check2) {
+  let areEqual = true;
```
Comment (davidpmccormick) [inline updown/webapp/utils.js:2]: how about `return check1.every((value, index) => value === check2[index])` ?

**Review by davidpmccormick — APPROVED** [review]
Looks good.

File: updown/webapp/expected-checks.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
+const expectedChecks = [
```
Comment (kenoir) [inline updown/webapp/expected-checks.js:1]: Nice

**Comment by kenoir** [comment]
@gestchild I can see the new checks in the updown.io console, but you should also be able review them via API.

**Review by jamesgorrie — APPROVED** [review]

**Pull request merged**